src/platform/forms-system/src/js/components/Telephone.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
```

Comment:
I feel like this shouldn't live in the forms-system, but it looks like we don't have a place for platform-wide components.

Reply:
Not tagging him here — y'all probably know better when to loop him in — but perhaps Rian Fowler can advise?

Reply:
I do plan on closing this PR, and opening a new PR in the VFST repo. I don't know how long it'll take to get approved though.